pkg/dependency/go.go

```diff
@@ -75,11 +82,16 @@ func (g Go) GetDependencyVersion(version string) (DepVersion, error) {
 
 	licenses, err := g.licenseRetriever.LookupLicenses("go", depURL)
 	if err != nil {
-		return DepVersion{}, fmt.Errorf("could not get retrieve licenses: %w", err)
+		return DepVersion{}, fmt.Errorf("could not convert to SemVer: %w", err)
```

Comment:
This doesn't look like it should have changed.

Reply:
@ryanmoran good catch. Fixed it